src/common/ErrorPage/styled.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+
+export const ErrorContainer = styled.div`
+    text-align: center;
+    background-color: ${({ theme }) => theme.colors.Whisper};
```

Comment:
Mam wrażenie, że background dla Errora powininen być jako Mercury, a chyba najlepiej byłoby tu dać transparent i niech po prostu komponent Errora, weźmie sobie tło z niższej warstwy Main ;)

Reply:
Patrząc na figmę wydaje mi się, że ten Whisper jest poprawny. Mercury to chyba tło figmy a nie naszej apki.